opencraft/urls.py

```diff
@@ -41,3 +41,9 @@
     url(r'^favicon\.ico$', RedirectView.as_view(url='/static/img/favicon/favicon.ico', permanent=False)),
     url(r'^$', views.IndexView.as_view(), name='index'),
 ]
+
+if settings.DEBUG:
+    import debug_toolbar
+    urlpatterns = [
```

Comment:
Just use `+=`?

Reply:
Ok, I tested and it works the same. I squashed the commits and force-pushed to keep it at 1 commit.

Reply:
I had to revert the `+=` because tests failed. It seems it's important the debug URLs come _before_ the other URLs (https://github.com/jazzband/django-debug-toolbar/issues/529#issuecomment-314851121).